src/find_controls.cpp

```diff
@@ -168,7 +167,7 @@ std::ostream& operator<<(std::ostream& os, const FindEnforce& val)
 
 boost::optional<std::vector<solver::Id>> GetEnvFindOnlySolver()
 {
-    static const auto once = GetEnvFindOnlySolverImpl();
+    const auto once = GetEnvFindOnlySolverImpl();
```

Comment:
The name "`once`" in incorrect and  should be changed.

[Notice] This would negatively affect performance. The overall effect is expected to be small, however.

Reply:
renamed, minor perf patch

Reply:
[resolved]